File: Unidad 3/C#/P3_U3.cs
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Dia
    {
        static void Main(string[] args)
        {
            int dia;
            Console.WriteLine("Programa para los dias de la semana.");
            Console.WriteLine("Introdusca un número para saber que día es: ");
            dia = Convert.ToInt32(Console.ReadLine());
            switch (dia)
            {
                case 1:
                    Console.WriteLine("Es lunes");
                    break;
                case 2:
                    Console.WriteLine("Es martes");
                    break;
                case 3:
                    Console.WriteLine("Es miercoles");
                    break;
                case 4:
                    Console.WriteLine("Es jueves");
                    break;
                case 5:
                    Console.WriteLine("Es viernes");
                    break;
                case 6:
                    Console.WriteLine("Es sabado");
                    break;
                case 7:
                    Console.WriteLine("Es domingo");
                    break;
            }
        }
    }
}

File: Unidad 3/JAVA/P3_U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p3_u3;

import java.util.Scanner;

/**
 *
 * @author LaboratorioLTI2
 */
public class P3_U3 {

    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int dia;
        System.out.println("Programa para los dias de la semana.");
        System.out.println("Introdusca un número para saber que día es: ");
        dia=sc.nextInt();
                switch(dia){
                    case 1:
                        System.out.println("Es lunes");
                        break;
                    case 2:
                        System.out.println("Es martes");
                        break;
                    case 3:
                        System.out.println("Es miercoles");
                        break;
                    case 4:
                        System.out.println("Es jueves");
                        break;
                    case 5:
                        System.out.println("Es viernes");
                        break;
                    case 6:
                        System.out.println("Es sabado");
                        break;
                    case 7:
                        System.out.println("Es domingo");
                        break;
                }
    }
}
